diff --git a/__init__.py b/__init__.py
index 3bc66f9..ac6956b 100644
--- a/__init__.py
+++ b/__init__.py
@@ -3,25 +3,24 @@ from pupa.scrape import Jurisdiction
 from .people import PersonScraper
 
 
-class Example(Jurisdiction):
-    jurisdiction_id = 'ocd-jurisdiction/country:us/state:ex/place:example'
-    name = 'Example Legislature'
-    url = 'http://example.com'
+class Albequerque(Jurisdiction):
+    jurisdiction_id = 'ocd-jurisdiction/country:us/state:nm/place:albequerque/council'
+    name = 'Albequerque City Council'
+    url = 'http://www.cabq.gov/council/'
     terms = [{
-        'name': '2013-2014',
-        'sessions': ['2013'],
+        'name': '2013-2015',
         'start_year': 2013,
-        'end_year': 2014
+        'end_year': 2015,
+        'sessions': ['2013'],
     }]
     provides = ['people']
     parties = [
-        {'name': 'Independent' },
-        {'name': 'Green' },
-        {'name': 'Bull-Moose'}
+        {'name': 'Democratic' },
+        {'name': 'Republican' },
     ]
     session_details = {
         '2013': {'_scraped_name': '2013'}
